test/smoke/process-usage/src/test/fixtures.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+const timewindow = require('abacus-timewindow');
+
+const { times } = require('underscore');
+
+const { fixturesCfg } = require('./env-config');
+// The scaling factor of each time window for creating the date string
+// [Second, Minute, Hour, Day, Month]
+const _slack = () =>
+  /^[0-9]+[MDhms]$/.test(fixturesCfg.slack)
+    ? {
+      scale: fixturesCfg.slack.charAt(fixturesCfg.slack.length - 1),
+      width: fixturesCfg.slack.match(/[0-9]+/)[0]
+    }
+    : {
+      scale: timewindow.dimension.min,
+      width: 10
+    };  
+
+const _initWindows = (win, dimension) => {
+  const windows = [win];
+
+  if(fixturesCfg.windowsSizes && fixturesCfg.windowsSizes[dimension])
+    times(fixturesCfg.windowsSizes[dimension] - 1, () => windows.push(null));
+
+  else {
+    const timeWindows = timewindow.timeWindowsSizes(_slack(), fixturesCfg.windowsSizes);
+    times(timeWindows.getWindows(dimension).length - 1, () => windows.push(null));
+  }
+
+  return windows;
+};
+
+const buildExpectedWindows = (summary, quantity) => {
+
+  const addProperty = (key, value, obj) => {
+    if (value)
+      obj[key] = value;
+  };
+
+  const win = {};
+  addProperty('quantity', quantity, win);
+  addProperty('summary', summary, win);
+
+  return [
+    [null],
+    [null],
+    [null],
+    _initWindows(win, timewindow.dimension.day),
+    _initWindows(win, timewindow.dimension.month)
+  ];
+};
+
+// Initial expected usage report for the test organization
+const createExpectedInitialReport = (orgID, summaries, quantites) => ({
```

Comment:
The initial expected report is static. We can move it to an external file and require it. I don't think we gain much by having it inline and building it with functions.

Reply:
We may move it in separate file but we have to move with it some utility functions that build windows fields in reporting (_buildExpectedWindows). 

If we maintain it in external file every change of measures or number of usages send to Abacus is going to result in two changes - one in the test file itself and one in that external file.

Reply:
We have been discussing to subtract current and previous reports. So that, we are going to move to external file expected diff report.